documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationSimple.adoc

```diff
@@ -1,4 +1,4 @@
-Simple authorization in Strimzi uses the `AclAuthorizer` plugin, the default Access Control Lists (ACLs) authorization plugin provided with Apache Kafka.
+Simple authorization in Strimzi uses the `AclAuthorizer` and `StandardAuthorizer` plugins, the default Access Control Lists (ACLs) authorization plugin provided with Apache Kafka.
```

Comment:
```suggestion
For simple authorization, Strimzi uses Kafka's built-in authorization plugins: the `StandardAuthorizer` for KRaft mode and the `AclAuthorizer` for ZooKeeper-based cluster management.
```

Reply:
Might be useful to make the distinction in this section.

Reply:
What do you mean with distinction here?

Reply:
Oh, I guess that was an explanation tot he suggestion above which looks good. Thanks.